js/internal-infrastructure/s3-redirect-generator/src/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2021 by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+// eslint-disable-next-line import/no-unresolved
```

Comment:
I wish that I didn't have to put these here, but I couldn't get the files to resolve without them when using this as a binary from the docs. :man_shrugging: